[Merged by Bors] - Ipv6 bootnodes update

AgeManning opened the pull request:
We now officially have ipv6 support. The mainnet bootnodes have been updated to support ipv6. This PR updates lighthouse's internal bootnodes for mainnet to avoid fetching them on initial load.

AgeManning committed:
Update ENR to support ipv6

divagant-martian commented:
can connect to the two new bootnodes just fine :zap: 

but you forgot to update the tests :stuck_out_tongue:

AgeManning committed:
Update bootnode test

AgeManning commented:
Yeah, i changed that test. Not sure what its testing, but I think updating ENRs we shouldn't have to update this random test with the exact value. 

Changed the test, to just make sure the count is the same.

AgeManning committed:
fmt

AgeManning committed:
Add new ipv6 bootnodes

AgeManning committed:
fmt

one-three-three-seven commented on common/eth2_network_config/built_in_network_configs/mainnet/boot_enr.yaml:
```diff
@@ -1,6 +1,8 @@
-- enr:-Jq4QN_YBsUOqQsty1OGvYv48PMaiEt1AzGD1NkYQHaxZoTyVGqMYXg0K9c0LPNWC9pkXmggApp8nygYLsQwScwAgfgBhGV0aDKQtTA_KgEAAAD__________4JpZIJ2NIJpcISLosQxiXNlY3AyNTZrMaEDBJj7_dLFACaxBfaI8KZTh_SSJUjhyAyfshimvSqo22WDdWRwgiMo
+- enr:-Le4QPUXJS2BTORXxyx2Ia-9ae4YqA_JWX3ssj4E_J-3z1A-HmFGrU8BpvpqhNabayXeOZ2Nq_sbeDgtzMJpLLnXFgAChGV0aDKQtTA_KgEAAAAAIgEAAAAAAIJpZIJ2NIJpcISsaa0Zg2lwNpAkAIkHAAAAAPA8kv_-awoTiXNlY3AyNTZrMaEDHAD2JKYevx89W0CcFJFiskdcEzkH_Wdv9iW42qLK79ODdWRwgiMohHVkcDaCI4I
+- enr:-Le4QLHZDSvkLfqgEo8IWGG96h6mxwe_PsggC20CL3neLBjfXLGAQFOPSltZ7oP6ol54OvaNqO02Rnvb8YmDR274uq8ChGV0aDKQtTA_KgEAAAAAIgEAAAAAAIJpZIJ2NIJpcISLosQxg2lwNpAqAX4AAAAAAPA8kv_-ax65iXNlY3AyNTZrMaEDBJj7_dLFACaxBfaI8KZTh_SSJUjhyAyfshimvSqo22WDdWRwgiMohHVkcDaCI4I
+- enr:- Le4QH6LQrusDbAHPjU_HcKOuMeXfdEB5NJyXgHWFadfHgiySqeDyusQMvfphdYWOzuSZO9Uq2AMRJR5O4ip7OvVma8BhGV0aDKQtTA_KgEAAAAAIgEAAAAAAIJpZIJ2NIJpcISLY9ncg2lwNpAkAh8AgQIBAAAAAAAAAAmXiXNlY3AyNTZrMaECDYCZTZEksF-kmgPholqgVt8IXr-8L7Nu7YrZ7HUpgxmDdWRwgiMohHVkcDaCI4I
```
There is an unnecessary space after "enr:-"

AgeManning replied:
Yep, thanks for the pickup

AgeManning committed:
Remove space

paulhauner reviewed (approved):
Looks good to me, happy to merge after we have @antondlr's blessing.

paulhauner commented:
bors r+

divagant-martian commented:
this makes me happy :smiling_face_with_tear:
